tools/design-tokens/generate-color-palette/src/main.ts

```diff
@@ -30,14 +30,14 @@ export function main(): void {
     out: { type: 'string' },
   }).argv;
 
-  const colorFileContents = readFileSync(colorFile, {
+  const colorFileContents = readFileSync(colorFile!, {
     encoding: 'utf-8',
   });
   const paletteSource = parse(colorFileContents);
   const aliases = generatePaletteAliases(paletteSource);
   const paletteOutput = stringify(aliases);
   writeFileSync(
-    out,
+    out!,
```

Comment:
I would do it with an if check and throw an error that you have to provide them or use the default to provide a sensible default for it.